editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -2579,7 +2579,7 @@ void CPUParticles3DGizmoPlugin::redraw(EditorNode3DGizmo *p_gizmo) {
 GPUParticles3DGizmoPlugin::GPUParticles3DGizmoPlugin() {
 	Color gizmo_color = EDITOR_DEF("editors/3d_gizmos/gizmo_colors/particles", Color(0.8, 0.7, 0.4));
 	create_material("particles_material", gizmo_color);
-	gizmo_color.a = 0.1;
+	gizmo_color.a = MAX((gizmo_color.a - 0.2) * 0.02, 0.0);
```

Comment:
I think we should look into harmonizing this behavior across all 3D gizmos. It can be done in a future PR, but we should make sure to avoid inconsistencies in this regard when Godot 4.0 is released.

Reply:
That's actually a very valid point.
I will take care about that as my next task 👍

Reply:
That's very true. If this PR is done, I will look into it and open a PR.